resources/views/livewire/portal/shop/checkout.blade.php

```diff
@@ -0,0 +1,108 @@
+        <x-card :title="__('Edit delivery address')">
+            <x-flux::address.address :only-postal="true" />
+            <div class="flex flex-wrap gap-4 pt-4">
+                @foreach(auth()->user()->contact->addresses as $address)
+                    <div
+                        class="cursor-pointer rounded-md bg-secondary-100 p-2"
+                        x-on:click="$wire.loadAddress({{ $address->id }})"
+                        x-bind:class="$wire.address.id === {{ $address->id }} && 'ring-2 ring-offset-2 ring-primary-500'"
+                    >
+                        {!! implode('<br />', $address->postal_address)  !!}
+                    </div>
+                @endforeach
+                <div
+                    class="cursor-pointer rounded-md bg-secondary-100 p-2 flex justify-center items-center flex-col"
+                    x-on:click="$wire.loadAddress()"
+                    x-bind:class="$wire.address.id === null && 'ring-2 ring-offset-2 ring-primary-500'"
+                >
+                    <x-icon class="w-5 h-5" name="plus" />
+                    <div>
+                        {{  __('Add new address') }}
+                    </div>
+                </div>
+            </div>
+            <x-slot:footer>
+                <div class="flex justify-end gap-1">
+                    <x-button x-on:click="close()">{{ __('Cancel') }}</x-button>
+                    <x-button wire:click="saveDeliveryAddress().then((success) => success ? close() : null)" primary>{{ __('Save') }}</x-button>
+                </div>
+            </x-slot>
+        </x-card>
+    </x-modal>
+    <x-card :title="__('Terms And Conditions')">
+        <div class="flex items-center gap-1.5">
+            <x-checkbox wire:model.boolean="termsAndConditions" />
+            <div>
+                {{ __('I accept the terms and conditions') }}
+            </div>
+            <div wire:click="loadTermsAndConditions().then((text) => {document.getElementById('terms-and-conditions').innerHTML = text; $openModal('terms-and-conditions')})" class="cursor-pointer text-primary-500 underline">
+                {{ __('Read terms and conditions') }}
+            </div>
+        </div>
+    </x-card>
+    <div class="flex flex-col sm:flex-row gap-4 justify-between">
+        <x-card :title="__('Invoice Address')">
+            <p>
+                {!! implode('</p><p>', auth()->user()->contact->invoiceAddress->postal_address)  !!}
+            </p>
+        </x-card>
+        <x-card :title="__('Delivery Address')">
+            <x-slot:action>
+                <x-button xs x-on:click="$openModal('edit-delivery-address')" :label="__('Edit delivery address')" />
+            </x-slot:action>
+            {!! implode('</p><p>', $this->deliveryAddress->postalAddress())  !!}
+        </x-card>
+    </div>
+    <x-card>
+        <div class="flex flex-col gap-4">
+            @if(auth()->user()?->contact?->priceList?->is_net)
```

Comment:
why only add desired delivery date if priceList->is_net?

Reply:
in case of a "regular end customer" i dont want to show a desired delivery date, its some kind of standard for b2b but not for b2c